cpp/src/Glacier2/RoutingTable.cpp

```diff
@@ -58,7 +58,7 @@ Glacier2::RoutingTable::add(const ObjectProxySeq& unfiltered, const Current& cur
             continue;
         }
 
-        if(!_verifier->verify(prx))
+        if(!_verifier->verify(*prx))
```

Comment:
Note that at this point prx is not null so the prx in the EvictorEntry created below will be not null.

Reply:
Yes, that is why unwrap it with `*` and updated verify to taken a non-optional proxy